tests/tests_suite_test.go

```diff
@@ -44,6 +44,10 @@ var (
 )
 
 func getDir() string {
+	// if inside Docker container, don't return nothin'
```

Comment:
My preference is to follow [12-factor](http://12factor.net/) practices and specify it in an env var.